relentless/simulate/simulate.py

```diff
@@ -109,3 +112,184 @@ class Data:
     @abc.abstractmethod
     def __call__(self, sim):
         pass
+
+class Potentials:
+    """Combination of multiple potentials.
+    """
+    def __init__(self):
+        self._pair = PairPotentialTabulator(None,None)
```

Comment:
It might be good to give a convenience argument to the constructor `pair_potentials`:
```suggestion
    def __init__(self, pair_potentials=None):
        self._pair = PairPotentialTabulator(None,None,potentials=pair_potentials)
```